src/main/scala/io/pubcrawler/stout/db/TableDefinitions.scala

```diff
@@ -26,14 +25,14 @@ trait TableDefinitions {
     def * = (id.?, ownerId) <> (Route.tupled, Route.unapply)
   }
 
-  val routes: TableQuery[Routes] = TableQuery[Routes]
+  val _routes: TableQuery[Routes] = TableQuery[Routes]
 
   class Crawls(tag: Tag) extends Table[Crawl](tag, "crawls") {
     def id = column[Int]("crawl_id", O.PrimaryKey, O.AutoInc)
     def title = column[String]("title")
     def ownerId = column[Int]("owner_id")
     def routeId = column[Int]("route_id")
-    def dateTime = column[Time]("date_time")
+    def dateTime = column[LocalDateTime]("date_time")
```

Comment:
We may need to have a discussion whether we want `LocalDateTime` or `ZonedDateTime`.